universal-application-tool-0.0.1/app/services/applicant/ApplicantService.java

```diff
@@ -0,0 +1,28 @@
+package services.applicant;
+
+import com.google.common.collect.ImmutableSet;
+import java.util.concurrent.CompletionStage;
+import models.Applicant;
+import services.ErrorAnd;
+
+/** Defines the interface facade for Applicant service */
+interface ApplicantService {
+
+  /**
+   * Performs a set of updates to the applicant's ApplicantData. Updates are atomic i.e. if any of
+   * them fail validation none of them will be written. programId is used to construct the {@code
```

Comment:
question: Why not use `{@link ReadOnlyApplicantService}` here? Just trying to learn how to use javadoc fanciness.

Reply:
Out of sheer laziness and not wanting to import it 😂.. It's a new day. I'll change it 👍🏻

Reply:
done